graphql/student/mutations.ts

```diff
@@ -160,7 +202,73 @@ export async function updateStudent(context: GraphQLContext, student: Student, u
     await updateSessionUser(context, userForStudent(res));
 
     log.info(`Student(${student.id}) updated their account with ${JSON.stringify(update)}`);
+    return res;
 }
+
+async function studentRegisterPlus(data: StudentRegisterPlusInput, ctx: GraphQLContext): Promise<{ success: boolean; reason: string }> {
+    const log = logInContext('Student', ctx);
+    const { email, register, activate, screen } = data;
```

Comment:
Uh, we should validate and normalize the email with

email = validateEmail(email);

Also this might be a good candidate for an integration test :)